pkg/multihash/multihash_test.go

```diff
@@ -0,0 +1,69 @@
+package multihash_test
+
+import (
+	"testing"
+
+	"github.com/trustbloc/orb/pkg/multihash"
+	"github.com/trustbloc/orb/pkg/store/cas"
+
+	"github.com/hyperledger/aries-framework-go/component/storageutil/mem"
+	"github.com/stretchr/testify/require"
+)
+
+func TestToV1CID(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		v1CID, err := multihash.ToV1CID("uEiA329wd6Aj36YRmp7NGkeB5ADnVt8ARdMZMPzfXsjwTJA")
+		require.NoError(t, err)
+		require.Equal(t, "bafkreibx3pob32ai67uyizvhwndjdydzaa45ln6acf2mmtb7g7l3epateq", v1CID)
+	})
+	t.Run("Fail to decode multibase-encoded multihash", func(t *testing.T) {
+		v1CID, err := multihash.ToV1CID("")
+		require.EqualError(t, err, "failed to decode multibase-encoded multihash: "+
+			"cannot decode multibase for zero length string")
+		require.Empty(t, v1CID)
+	})
+	t.Run("Fail to parse the decoded multibase value as a multihash", func(t *testing.T) {
+		v1CID, err := multihash.ToV1CID("u2ouBBsyLDedeYciUmaihjWUmWhA3LVkruPZwSk7EeHWWqUZF")
```

Comment:
@troyronda I took (`u2ouBBsyLDedeYciUmaihjWUmWhA3LVkruPZwSk7EeHWWqUZF`) directly from https://trustbloc.github.io/activityanchors/#example-1-operation-batch but it doesn't seem to be valid, as you can see from the unit test.

(`uEiA329wd6Aj36YRmp7NGkeB5ADnVt8ARdMZMPzfXsjwTJA` is also used in the example, but it seems to be valid - you an see that in the first test here)